commands/builds/cache-purge.js

```diff
@@ -2,8 +2,19 @@
 
 const cli = require('heroku-cli-util')
 const co = require('co')
+const util = require('util')
+
+const confirmMsg = `WARNING: This will delete the build cache for %s.
+Buildpacks use that cache to store information across builds..
+Clearing your build cache can have unintended side effects, such as updating your language version.`
```

Comment:
Should there be something about "unless you're explicitly specifying one" to prevent everyone from being concerned of consequences?

Reply:
Good idea. I've done that.